discovery-infra/install_cluster.py

```diff
@@ -155,6 +166,12 @@ def main():
         type=str,
         default='assisted-service'
     )
+    parser.add_argument(
```

Comment:
what will happened if we have couple of clusters with the same name?

Reply:
I guess cluster-id should be preffered

Reply:
this is problem doesn't necessarily related to this pr since this question is relevant also for current state.

Reply:
I will fix this in a different pull request